token/cli/src/clap_app.rs

```diff
@@ -591,6 +591,15 @@ pub fn app<'a, 'b>(
                 .possible_values(&["json", "json-compact"])
                 .help("Return information in specified output format"),
         )
+        .arg(
+            Arg::with_name("program_2022")
+                .short("2")
```

Comment:
This PR is ready to go in except for this. We really don't want to use a number for a short arg like this, since it manifests as `<command> -2` and that could easily be mistaken for a numerical negative two.

Can we just drop the `.short` here?

Reply:
Sure 🫡
done